src/pages/AppDashboard.vue

```diff
@@ -121,34 +121,39 @@ async function fetchAndPopulateCharts() {
   updateChart(periodicUsage.data)
 }
 
+function getAllowedLimit(limit: number) {
+  return limit || MAX_ALLOWED_APP_LIMIT_IN_BYTES
```

Comment:
When will `data.storage` and `data.bandwidth` be missing from the api response?

Reply:
Removed here https://github.com/arcana-network/developer-dashboard/pull/135/commits/134e776175d244327e99ab2630bdd290597ff539

Reply:
@ajithranka no, I just confirmed, these fields won't be missing from api response, it'll always be there

Reply:
Then I think we can remove the fallback limit from the frontend.